### ISSUE TITLE
tag cload font

### ISSUE THREAD

**Issue by mohtasham1988** [AnyChart/AnyChart-Android#25]
![20180806_135037](https://user-images.githubusercontent.com/22011648/43708590-0889054c-9980-11e8-9553-eac2460ffa13.jpg)

hi 
i have problem when i chenge font to persian 

please help

**Comment by mohtasham1988** [comment]
@Shestac92 
yes, the problem is that tags are overlapping each other

**Comment by Shestac92** [comment]
@mohtasham1988 
yes, now I see clearly the problem. I will show it to our dev team and hopefully, soon we will provide the fix for this. I will notify you when the fix or workaround becomes available.
Thank you for your report!
